tools/gfx_tools/decompose.py

```diff
@@ -304,15 +307,15 @@ def get_all_data(self, tileset_dirname, delete_pathname):
             os.stat(self.tileset_pathname)
         except KeyError:
             print("cannot find a directory {}".format(self.tileset_pathname))
-            exit -1
+            sys.exit(-1)
```

Comment:
@mlangsdorf It's 255 in the end, is it what you needed or should it be just `1`?